### ISSUE TITLE
Pokestops not appearing and Pokemon not appearing

### ISSUE THREAD

**Issue by JShenpai** [finndev/PokeBuddy#427]
Pokestops not appearing and Pokemon not appearing, what's happening?

**Comment by Giammaofwar** [comment]
Had the same, disable your antivirus or make an exception

**Comment by will127** [comment]
yup whent though my antivirus setting and had to retrust it cuz of the update  FYI guys  now its working though norton  again

**Comment by JShenpai** [comment]
Huh, I don't have antivirus. Seems to be working now though, weird. Thanks for the help guys.
